src/index.js

```diff
@@ -39,6 +53,27 @@ const addTodo = (description) => {
   renderTodo(todo);
 };
 
+const toggleDone = (key) => {
+  const index = todoItems.findIndex((item) => item.index === Number(key));
+  todoItems[index].completed = !todoItems[index].completed;
+  renderTodo(todoItems[index]);
+};
+
+const deleteTodo = (key) => {
+  // find item in the todo items array
+  const index = todoItems.findIndex((item) => item.index === Number(key));
+
+  // create new object with a deleted property
+  const todo = {
+    deleted: true,
+    ...todoItems[index],
+  };
+
+  // remove todo item from array
+  todoItems = todoItems.filter((item) => item.index !== Number(key));
+  renderTodo(todo);
```

Comment:
- [ ] as the requirements, it would be better when removing a task from the todo list you should update the indexes of the other tasks so please try to do that

Reply:
Hi @youmari, 
Thank you for the review. 
When I remove a task, the other tasks indices are updated. Kindly view the video below that demonstrates this. 

https://user-images.githubusercontent.com/92020713/172264544-4a542b07-117a-476b-9c5e-4d04dcd0fb96.mov